src/lib/tvgSceneImpl.h

```diff
@@ -65,7 +65,7 @@ struct Scene::Impl
         Compositor* cmp = nullptr;
 
         //Half translucent. This condition requires intermediate composition.
-        if ((opacity < 255 && opacity > 0) && (paints.count > 1)) {
+        if ((opacity < 255 && opacity > 0) && (paints.count > 0)) {
```

Comment:
if the scene has just one simple share, composition is not required, 
it would be better if condition check whether child is another scene or not.

Reply:
Is there a way to know if a child is a scene or another shape or picture?
Since we are using the `fno-rtti` option, we cannot use `dynamic_cast` and class `typeid`
So there seems to be no way to check the child's class type. is that right?
I don't think of a good way.
Another way is to create a type get function.
```
enum PaintType {Shape, Scene, Picture};
PaintType type(Paint*);
```
Please share any good idea.

Reply:
@JSUYA I think we can put a type info(with accessing it) in the internal Paint member.